.config/tsaoptions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\mono",
```

Comment:
we should use a different area path here since we don't really own this repo. @slluis do you have an area path and notification alias we could use?